index.html

```diff
@@ -72,14 +72,14 @@ <h5 class="paragraph purpleColor">LET'S CONNECT</h5>
     </section>
 
     <section class="flexContent">
-        <div class="project">
+        <div class="project border">
             <aside class="aside aside-1">
                 <img src="images/img2.jpg" alt="" class="fullPic">
             </aside>
             <aside class="aside aside-2">
                 <h3 class="subtitle">Alpha Haiti</h3>
```

Comment:
https://github.com/MenendezON/Portfolio/blob/f8e48ea186fa2a3cdfca44b2c9e4a2a31fe41a64/index.html#L88-L92

- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be in a list `ul` and each of them a `li`. :+1: 

https://github.com/MenendezON/Portfolio/blob/f8e48ea186fa2a3cdfca44b2c9e4a2a31fe41a64/index.html#L140

- [ ] The button (e.g., "See project") is a link `a` or `button` tag, and kindly follow the button styling: Active, Hover and Pressed (see design). :+1: 

https://github.com/MenendezON/Portfolio/blob/f8e48ea186fa2a3cdfca44b2c9e4a2a31fe41a64/index.html#L74-L96

- [ ] The section has the margins (top, bottom, left, right) defined in the design. The section uses the font and colors (background and font) defined for the template. :+1: 

- [ ] The main container of the projects should have a display: grid as per the project requirements. Kindly work on it.

Note: Kindly do the change for other projects also.